### PR TITLE
First testing phase

### PR DESCRIPTION
**Pull request by RadeKornjaca**
Contains Cucumber scenarios for displaying events and places in application, also it has, by default, an user logged in on every each of them.

### CODE REVIEW

File: features/step_definitions/signup_login_steps.rb [hunk 1]
```diff
@@ -1,15 +1,13 @@
 # features/step_definitions/signup_login_steps.rb
 
 
-Given(/^user is signed in with username "(.*?)", email "(.*?)" and password "(.*?)"$/) do |username, email, password|
+Given(/^User is signed in with username "(.*?)", email "(.*?)" and password "(.*?)"$/) do |username, email, password|
```
Comment (mimimalizam) [inline features/step_definitions/signup_login_steps.rb]: `user is signed up`

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -1,14 +1,16 @@
+<p id="notice"><%= notice %></p>
+
 <h1>Listing events</h1>
 
 <div class="row">
 	<% @events.each do |event| %>
-		<div class="col-xs-6 col-lg-4">
+		<div id="<%= 'event_' + event.id.to_s %>" class="col-xs-6 col-lg-4">
```
Comment (darkofabijan) [inline app/views/events/index.html.erb]:
``` ruby
<%= content_tag(:div, :id => "event_" + event.id.to_s, :class => "col-xs-6 col-lg-4") do %>
...
<% end %>
```

``` ruby
<%= content_tag_for(:div, event, :class => "col-xs-6 col-lg-4") do %>

<% end %>
```

File: app/views/places/index.html.erb [hunk 3]
```diff
@@ -13,7 +15,7 @@
 
   <tbody>
     <% @places.each do |place| %>
-      <tr>
+      <tr id="<%= 'place_' + place.id.to_s %>">
```
Comment (darkofabijan) [inline app/views/places/index.html.erb]: http://api.rubyonrails.org/classes/ActionView/Helpers/RecordTagHelper.html#method-i-content_tag_for

File: features/user_page.feature [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    Given user do not exist
+    When he chooses to sign in
+    And he fills up form with his email and password
+    And clicks Sign in button
+    Then he should stay on the Sign in page
+
+  Scenario: New user signs up
+    Given user do not exist
+    When he presses the Sign Up button
+    Then he should see sign up form
```
Comment (darkofabijan) [inline features/user_page.feature:23]: Switch with one step that opens a sign up page.

File: features/events_page.feature [hunk 5]
```diff
@@ -1,17 +1,47 @@
-# features/events_page.feature
 
 Feature: Events Page
 
-	Background: Signing up and logging in
-		Given user is signed in with username "zika", email "zikin.mejl@gmail.com" and password "zikazika"
-		And User "zika" is logged in
+	Background: User is signed and logged in
```
Comment (darkofabijan) [inline features/events_page.feature]: User is logged in

File: features/events_page.feature [hunk 6]
```diff
@@ -1,17 +1,47 @@
...
+    Given user exists
+    And user is on the homepage
+    When he chooses to sign in
+    And he fills up form with his email and password
+    And clicks Sign in button
```
Comment (darkofabijan) [inline features/events_page.feature]: Given user is logged in

**Pull request merged**